doc/source/examples/analysis/custom_parallel_analysis.ipynb

```diff
@@ -0,0 +1,986 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "result = np.asarray(result).T\n",
+    "\n",
+    "labels = ['all', 'x-axis', 'y-axis', 'z-axis']\n",
+    "for col, label in zip(result, labels):\n",
+    "    plt.plot(col, label=label)\n",
+    "plt.legend()\n",
+    "plt.ylabel('Radius of gyration (Å)')\n",
+    "plt.xlabel('Frame');"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can also use Dask dashboard (with dask.distributed.Client) to examine how jobs are distributed along all the workers. So each green bar below represents a single-frame `radgyr`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<center>\n",
+    "<div style=\"width: 650px; text-align: center;\">\n",
+    "    \n",
+    "![Dask task stream](../images/parallel_analysis/per_frame_dask.png)\n",
+    "\n",
+    "</div>\n",
+    "</center>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Parallelization in a split-apply-combine fashion\n",
+    "\n",
+    "The aforementioned per-frame approach should normally be **avoided** because in **each** task, all the attributes (`AtomGroup`, `Universe`, and etc) need to be pickled. This might even takes more time than your lightweight analysis! Besides, e.g. in dask, a huge amount of time is needed overhead to build a comprehensive dask graph with thousands of tasks.\n",
+    "\n",
+    "So a simple split-apply-combine scheme for parallel trajectory analysis should normally be applied (See the figure below)--the trajectory is split into blocks, analysis is performed separately and in parallel on each block (\"apply\"), then results from each block are gathered and combined. \n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<center>\n",
+    "<div style=\"width: 650px; text-align: center;\">\n",
+    "    \n",
+    "![Split-apply-combine approach](../images/parallel_analysis/dask_sac.png)\n",
+    "</div>\n",
+    "</center>\n",
+    "\n",
+    "Image from (<a data-cite=\"fan_pmda_2019\" href=\"https://doi.org/10.25080/Majora-7ddc1dd1-013\">Shujie *et al.*, 2019</a>) used under CC-BY license."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We will show a simple illustration of split-apply-combine approach with dask below:"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Block analysis function"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "@dask.delayed\n",
```

Comment:
Can you just briefly explain that dask.delayed as a decorator here is the same as `dask.delayed(analyze_block)` or something like that?